src/EventListener/XssProtectionListener.php

```diff
@@ -42,6 +45,8 @@ public function onKernelResponse(ResponseEvent $e): void
             return;
         }
 
+        trigger_deprecation('nelmio/security-bundle', '3.3.1', 'The "%s" class is deprecated, use Content Security Policy without allowing "unsage-inline" scripts instead.', __CLASS__);
```

Comment:
```suggestion
        trigger_deprecation('nelmio/security-bundle', '3.4.0', 'The "%s" class is deprecated, use Content Security Policy without allowing "unsage-inline" scripts instead.', __CLASS__);
```

I think we can maybe trigger this deprecation from the constructor, since we are deprecating the class.